kubernetes-python/__main__.py

```diff
@@ -0,0 +1,16 @@
+import pulumi
+from pulumi_kubernetes.apps.v1 import Deployment
+
+app_labels = { "app": "nginx" }
+
+deployment = Deployment("nginx",
```

Comment:
Also recommend bumping `"nginx"` down to the next line so that it follows PEP-8.

```python3
deployment = Deployment(
    "nginx",
    spec={
        "selector": {"match_labels": app_labels},
        "replicas": 1,
        "template": {
            "metadata": {"labels": app_labels},
            "spec": {"containers": [{"name": "nginx", "image": "nginx"}]}
        }
    })
```